specification/columns/servicesubcategory.md

```diff
@@ -2,10 +2,12 @@
 
 The Service Subcategory is a secondary classification of the Service Category for a [*service*](#glossary:service) based on its core function. The Service Subcategory (in conjunction with the Service Category) is commonly used for scenarios like analyzing spend and usage for specific workload types across providers and tracking the migration of workloads across fundamentally different architectures.  
 
-* The ServiceSubcategory column MUST be present in a FOCUS dataset and MUST NOT be null.
-* This column is of type String and MUST be one of the allowed values.
-* Each Service Subcategory value MUST have one and only one Service Category parent.
-* Though a given *service* can have multiple purposes, each *service* SHOULD have one and only one Subcategory that best aligns with its primary purpose.
+The ServiceSubcategory column adheres to the following requirements:
+
+* ServiceSubcategory is RECOMMENDED to be present in a FOCUS dataset and MUST NOT be null.
+* ServiceSubcategory is of type String and MUST be one of the allowed values.
+* Each ServiceSubcategory value MUST have one and only one ServiceCategory parent.
```

Comment:
just a suggestion

```suggestion
* Each ServiceSubcategory value MUST have one and only one parent ServiceCategory.
```

Reply:
Question: 
Could/should we specify that the **parent is defined by FOCUS**? Meaning that providers could conform to this requirement while continuously providing a ServiceSubcategory: "Search" with a ServiceCategory: "Web"  ;)

Reply:
I think that's a good point, Irena!

Reply:
```suggestion
* Each ServiceSubcategory value MUST have one and only one parent ServiceCategory as specified in the allowed values below.
```